src/sentry/web/client_config.py

```diff
@@ -224,6 +224,11 @@ def enabled_features(self) -> Iterable[str]:
             yield "relocation:enabled"
         if features.has("system:multi-region"):
             yield "system:multi-region"
+        # TODO @athena: remove this feature flag after development is done
+        # this is a temporary hack to be able to used flagpole in a case where there's no organization
+        # availble on the frontend
+        if self.last_org and features.has("organizations:scoped-partner-oauth", self.last_org):
```

Comment:
Are you planning on using organization conditions for this feature check? The `last_org` attribute could reflect an arbitrary organization depending on where the session last visited. If you want to have user level conditions for this flag you should pass the `actor` parameter. Right now you won't be able to use user attributes.

Reply:
I see. So I can pass actor and last_org but from flagpol yml ignore the org completely?

Reply:
oh I see an example above. Like organization:create, just with actor.